doc.js

```diff
@@ -340,6 +340,18 @@ define('doc', ['event'], function(event) {
 				return this;
 			},
 
+			'trigger' : function(event) {
+				this.each(function(el) {
+					if (!window.addEventListener) {
+						el.fireEvent("on" + event);
```

Comment:
On IE the events related to touch has a different syntax (https://github.com/elo7/events-amd/blob/master/events-amd.js). I don't think someone will use trigger for touch related events, but it'd be great if it's fixed or added to the doc explaining it won't work on IE 9 and IE 10.

@erikatakahara